pontoon/base/placeables.py

```diff
@@ -3,38 +3,32 @@
 
 class PythonFormatNamedString:
     parser = re.compile(
-        r"(%\([[\w\d!.,[\]%:$<>+\-= ]*\)[+|-|0\d+|#]?[.\d+]?[s|d|e|f|g|o|x|c|%])",
+        r"(%\([[\w\d!.,[\]%:$<>+\-= ]*\)[+-0\d+#]?[.\d+]?[sdefgoxc%])",
         re.IGNORECASE,
     )
 
 
 class PythonFormatString:
-    parser = re.compile(r"(\{{?[\w\d!.,[\]%:$<>+-= ]*\}?})")
+    parser = re.compile(r"({{?[\w!.,[\]%:$<>+\-= ]*}?})")
```

Comment:
Based on my reading of the [Python spec](https://docs.python.org/3/library/string.html#format-string-syntax), doubled `{{` and `}}` are parsed as standalone escaped characters.

Also, this is a really lax implementation that's also capturing e.g. most `FluentTerm` values as well.

```suggestion
    parser = re.compile(r"{{|}}|{[\w!.,[\]%:$<>+\-= ]*}")
```